wallet/src/de/schildbach/wallet/transactions/WalletMostRecentTransactionsObserver.kt

```diff
@@ -28,7 +27,6 @@ import org.bitcoinj.core.Transaction
 import org.bitcoinj.utils.Threading
 import org.bitcoinj.wallet.Wallet
 
-@ExperimentalCoroutinesApi
 class WalletMostRecentTransactionsObserver(private val wallet: Wallet) {
     fun observe(): Flow<Transaction> = callbackFlow {
```

Comment:
Perhaps at a later time, we could merge the functions of this class into `WalletObserver`.